JCDispatch/src/main/java/lib/nameParsing.java

```diff
@@ -0,0 +1,155 @@
+package lib;
+
+import com.auth0.jwt.internal.org.apache.commons.lang3.text.WordUtils;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+
+public class nameParsing {
+
+    public static String formatText( Object obj ) {
```

Comment:
`formatText` is a very general name, but this method does something very specific. Please rename this to better reflect what the method actually does.